job-server-python/src/python/test/apitests.py

```diff
@@ -56,6 +57,8 @@ class TestSJSApi(unittest.TestCase):
     def setUp(self):
         conf = SparkConf().setAppName('test').setMaster('local[*]')
         self.sc = SparkContext(conf=conf)
+        self.jvm = self.sc._gateway.jvm
+        java_import(self.jvm, "org.apache.spark.sql.*")
```

Comment:
@zeitos hmm, I thought I backed out all those changes, but guess not. I'll update to take that out, I'm not sure what went wrong there.